db/model/Gdoc/GdocBase.ts

```diff
@@ -565,29 +565,6 @@ export class GdocBase extends BaseEntity implements OwidGdocBaseInterface {
         }
     }
 
-    async loadRelatedCharts(): Promise<void> {
```

Comment:
Only GdocPosts have related charts, so I moved this to be subclass-only, which helps when it comes time to render the feed page.